lists.py

```diff
@@ -0,0 +1,55 @@
+#####################
```

Comment:
Ik zou de voorbeeld opdracht iets kleiner en makkelijker maken. Dus misschien één opdracht zodat het zo kort mogelijk is :)